src/extension/teradata/src/test/java/org/geoserver/terradata/ModuleStatusTest.java

```diff
@@ -1,25 +1,34 @@
 package org.geoserver.terradata;
 
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
 
 import java.util.Optional;
 import org.geoserver.platform.GeoServerExtensions;
 import org.geoserver.platform.ModuleStatus;
+import org.geotools.data.teradata.TeradataDataStoreFactory;
 import org.junit.Test;
-import org.springframework.context.ApplicationContext;
 import org.springframework.context.support.ClassPathXmlApplicationContext;
 
 public class ModuleStatusTest {
     @Test
     public void test() {
 
-        ApplicationContext context = new ClassPathXmlApplicationContext("applicationContext.xml");
+        TeradataDataStoreFactory fac = new TeradataDataStoreFactory();
+        Boolean expect;
+        if (fac.isAvailable()) {
+            expect = Boolean.TRUE;
+        } else {
+            expect = Boolean.FALSE;
+        }
+        try (ClassPathXmlApplicationContext context =
+                new ClassPathXmlApplicationContext("applicationContext.xml")) {
 
-        Optional<ModuleStatus> status =
-                GeoServerExtensions.extensions(ModuleStatus.class, context)
-                        .stream()
-                        .filter(s -> s.getModule().equalsIgnoreCase("gs-terradata"))
-                        .findFirst();
-        assertTrue(status.isPresent());
+            Optional<ModuleStatus> status =
+                    GeoServerExtensions.extensions(ModuleStatus.class, context)
+                            .stream()
+                            .filter(s -> s.getModule().equalsIgnoreCase("gs-terradata"))
```

Comment:
I think it's teradata, not terradata (single r vs double r). Maybe it's already in the pom files?

Reply:
That'll teach me to test without downloading the stupid jars to enable it